bindings/java/src/main/java/org/ucx/jucx/ucp/UcpWorkerParams.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.ucp;
+
+import java.nio.ByteBuffer;
+import java.util.BitSet;
+
+import org.ucx.jucx.UcxParams;
+import org.ucx.jucx.ucs.UcsConstants;
+
+public class UcpWorkerParams extends UcxParams {
+
+    private int threadMode;
+
+    private BitSet cpuMask = new BitSet();
+
+    private long events;
+
+    private ByteBuffer userData;
+
+    private int eventFD;
+
+    @Override
+    public UcpWorkerParams clear() {
+        super.clear();
+        threadMode = 0;
+        cpuMask = new BitSet();
+        events = 0;
+        userData = null;
+        eventFD = 0;
+        return this;
+    }
+
+    /**
+     * Requests the thread safety mode which worker and the associated resources
+     * should be created with. By default UCS_THREAD_MODE_SINGLE is used.
```

Comment:
need to remove `UCS_THREAD_MODE_SINGLE` mentioning from the docs since it's not part of API now.